plugins/source/datadog/client/spec.go

```diff
@@ -1,5 +1,7 @@
 package client
 
+import _ "embed"
+
 type Spec struct {
```

Comment:
It would be great to add the descriptions from https://hub.cloudquery.io/plugins/source/cloudquery/datadog/v4.2.8/docs?search=datadog#configuration-datadog-spec as comments here, so it will be propagated to the JSONSchema as well